src/main/kotlin/org/kamiblue/client/command/commands/ModifiedCommand.kt

```diff
@@ -0,0 +1,32 @@
+package org.kamiblue.client.command.commands
+
+import net.minecraft.util.text.TextComponentString
+import net.minecraft.util.text.event.ClickEvent
+import net.minecraft.util.text.event.HoverEvent
+import org.kamiblue.client.command.ClientCommand
+import org.kamiblue.client.util.text.MessageSendHelper.sendChatMessage
+
+object ModifiedCommand: ClientCommand(
+    name = "modified",
+    description = "View modified settings in a module"
+) {
+
+    init{
+        module("module"){
+            execute("List changed settings"){
+
+                for (setting in it.value.settingList.filter { it.value != it.defaultValue }){
+                    val component = TextComponentString("${setting.name} has been changed to ${setting.value}")
+                    // horrible, however this is mojang code that we are working on.
+                    component.style.clickEvent = ClickEvent(ClickEvent.Action.RUN_COMMAND, ";set ${it.value.name} ${setting.name.replace(" ", "")} ${setting.defaultValue}")
```

Comment:
If you know a better way of doing this then say, however you can't run code on click, just send commands in chat.

Reply:
Genius